_posts/2017-08-13-gsoc-karl-1.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Contributing to Babel: Three GitHub Issues at a Time"
+author: "Karl Cheng"
+slug: contributing-to-babel-three-github-issues-at-a-time
+---
+
+Getting to work your way around a new code base always poses its challenges, and Babel was no exception.
+
+I’ve been working with Babel as part of the Google Summer of Code 2017 programme, working to update Babel transforms and the Babylon parser to accommodate changes to specifications and implementing new features.
+
+Here’s a few things I’ve learnt from my adventures so far.
+
+## 1. Yes, communication is important
+
+To start off with getting to know the codebase better, I combed through the open issues list on Babel and found a relatively easy one ([issue #5728][#5728]) to deal with.
```

Comment:
It's actually referenced below:

https://github.com/babel/website/pull/1308/files#diff-3845b9c226ad4a8f0d99c33d3356f080R41